### DISCUSSION TITLE
Show video controls on tap

### DISCUSSION THREAD

**Discussion by NikhilSudhakaranVG** [deckerst/aves#833] [Ideas]
While playing videos, show video controls (mute, playback speed etc) on tap. Now it's  hidden under options--> video .

**Comment by deckerst** [comment]
Configure quick actions.

**Reply by NikhilSudhakaranVG** [reply]
Quick actions only available for thumbnails. 
I'm talking about video player.pls check screenshot. 
![Screenshot_20231205-162714_Aves.jpg](https://github.com/deckerst/aves/assets/46962641/7c8dd480-5cb3-469e-8465-2db637856c67)

**Reply by deckerst** [reply]
Nope. There are quick actions for the viewer too, via Settings > Viewer > Quick Actions.

**Reply by NikhilSudhakaranVG** [reply]
Thanks for your reply. Keep doing good work sir.
